src/components/Asset/AssetActions/Trade/Output.tsx

```diff
@@ -7,19 +7,21 @@ import styles from './Output.module.css'
 import { isValidNumber } from '@utils/numbers'
 import Decimal from 'decimal.js'
 import { FormTradeData } from './_types'
+import { usePool } from '@context/Pool'
 
 Decimal.set({ toExpNeg: -18, precision: 18, rounding: 1 })
 
 export default function Output({
   dtSymbol,
-  oceanSymbol,
+  baseTokenSymbol,
```

Comment:
with that new `usePool()` there would be no need to pass around all those props as you can get them from within this component with that hook under `poolInfo`